community/netsurf-fb/depends

```diff
@@ -0,0 +1,12 @@
+bison make
+gperf make
+libpng
+libressl
+libwebp
+linux-headers make
+perl make
+#sdl
```

Comment:
Commented dependency?

Reply:
It is possible to build this frontend with xcb-util or sdl interface.  
Maybe `#sdl - uncomment to build  with additional sdl interface` ?

Reply:
Sure. :+1: